wordpress-mobile/WordPress-Android
# [Jetpack Content Migration Flow] Integrate local migration with UI

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/17508

To test:

<details>

Enable the following flags in the `build.gradle`

```
        buildConfigField "boolean", "JETPACK_POWERED_BOTTOM_SHEET", "true"
        buildConfigField "boolean", "JETPACK_SHARED_LOGIN", "true"
        buildConfigField "boolean", "JETPACK_LOCAL_USER_FLAGS", "true"
        buildConfigField "boolean", "JETPACK_BLOGGING_REMINDERS_SYNC", "true"
        buildConfigField "boolean", "JETPACK_READER_SAVED_POSTS", "true"
        buildConfigField "boolean", "JETPACK_PROVIDER_SYNC", "true"
        buildConfigField "boolean", "JETPACK_MIGRATION_FLOW", "true"
```

</details>

**Generate some local-only content**:

Follow the steps in https://github.com/wordpress-mobile/WordPress-Android/pull/17398
#### Testing steps:

_Uninstall the Jetpack app or log out of Jetpack and delete all data._

1. Run the Jetpack app from this branch
2. Expect to be logged in automatically and see the list of your sites
3. Press continue on the next screens
4. Expect to see labels for local content in the post list

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
